framework/source/class/qx/io/remote/Rpc.js

```diff
@@ -15,67 +15,50 @@
    Authors:
      * Andreas Junghans (lucidcake)
      * Derrell Lipman (derrell)
+     * Christian Boulanger (cboulanger)
 
 ************************************************************************ */
 
 /**
- * Provides a Remote Procedure Call (RPC) implementation.
+ * Provides a Remote Procedure Call (RPC) version 2 implementation.
+ * See https://www.jsonrpc.org/specification
  *
  * Each instance of this class represents a "Service". These services can
  * correspond to various concepts on the server side (depending on the
- * programming language/environment being used), but usually, a service means
- * a class on the server.
- *
- * In case multiple instances of the same service are needed, they can be
- * distinguished by ids. If such an id is specified, the server routes all
- * calls to a service that have the same id to the same server-side instance.
+ * programming language/environment being used).
  *
  * When calling a server-side method, the parameters and return values are
- * converted automatically. Supported types are int (and Integer), double
- * (and Double), String, Date, Map, and JavaBeans. Beans must have a default
- * constructor on the server side and are represented by simple JavaScript
- * objects on the client side (used as associative arrays with keys matching
- * the server-side properties). Beans can also be nested, but be careful not to
- * create circular references! There are no checks to detect these (which would
- * be expensive), so you as the user are responsible for avoiding them.
+ * converted automatically to and from their JSON representation.
+ *
+ * Server-initiated calls
+ *
+ * Because of the client-server architecture of HTTP, which we use here, the
+ * bidirectional (peer-to-peer), real-time character of JSONRPC v2 cannot
+ * be fully implemented. Instead, any server-initiated remote procedure
+ * calls "piggy-back" on the responses to  client-initialed calls. If
+ * you want to continually listen to server calls, you must use polling
```

Comment:
It may be a "kludge" but I have use cases where websockets are not an option and a reverse transport via HTTP response is a rather elegant solution to that problem. But  I'll put it somewhere else.